src/Chainweb/Payload/RestAPI.hs

```diff
@@ -110,6 +115,7 @@ somePayloadDbVal v cid db = runIdentity $ do
 type PayloadGetApi_
     = "payload"
     :> Capture "BlockPayloadHash" BlockPayloadHash
+    :> QueryParam "blockHeight" BlockHeight
```

Comment:
Making this a path parameter (and possibly deprecating the query without it) may be a strong incentive for the use of the block height.